powerspikegg/serving/public/match_computation.proto

```diff
@@ -9,11 +9,43 @@ service MatchComputation {
 }
 
 message MatchComputationRequest {
-  string model_name = 1;
+    string model_name = 1;
 
-  game.leagueoflegends.MatchReference match = 2;
+    game.leagueoflegends.MatchReference match = 2;
+
+    // This id correspond to the global summoner id not the id of the summoner
+    // in the match
+    int32 summoner_id = 3;
 }
 
 message MatchComputationFeature {
-  string result = 1;
+  Statistics statistics = 1;
+}
+
+message StatisticFeature {
```

Comment:
Would be nice to document this message, to be sure what is coefficient and value. Are they the expected ones? Is the coefficient the standard score?.